### PR TITLE
Fix missing margin prop in several Labeled components

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

File: packages/ra-ui-materialui/src/input/SelectInput.tsx [hunk 1]
```diff
@@ -115,6 +115,7 @@ export const SelectInput = (props: SelectInputProps) => {
         label,
         loaded,
         loading,
+        margin = 'dense',
```
Comment (djhi) [inline packages/ra-ui-materialui/src/input/SelectInput.tsx:118]: Why a default here and not in the others ? Besides, I think the Form components already have this default.

**Review by djhi — COMMENTED** [review]
Thanks!

File: packages/ra-ui-materialui/src/input/SelectInput.tsx (hunk 1 above)
Comment (WiXSL) [reply packages/ra-ui-materialui/src/input/SelectInput.tsx:118]: I copy the default from the other inputs.
If you want I can remove all the defaults for margin?

Comment (WiXSL) [reply packages/ra-ui-materialui/src/input/SelectInput.tsx:118]: But I only see margin's default value only being set for most Inputs like that

Comment (djhi) [reply packages/ra-ui-materialui/src/input/SelectInput.tsx:118]: I forgot we did this, nvm :)

**Review by djhi — APPROVED** [review]

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**